src/linter/cache_test.go

```diff
@@ -101,6 +101,18 @@ function as_shape($x) {
   return $x;
 }
 
+/** With default */
+function withDefault($x = 100) {
+  return $x;
+}
+
+class WithDefaultMethod {
+  /** With default */
+  public function withDefault($x = 100) {
```

Comment:
Use different type of the default param here. String, for example.

Reply:
Done.